### ISSUE TITLE
[Feature Request]: Add option to exclude a post from the feed

### ISSUE THREAD

**Issue by s3nnet** [GetPublii/Publii#1202]
### Feature Description

A lot of blogs are part of a blog aggregator. OSBN.de in my case. To avoid the duplication of posts in case of a cooperation (disclaimer: s3nnet.de is cooperating with gnulinux.ch, which both are part of OSBN.de, an open-source blog aggregator)  I kindly request the possibility to exclude a post from the feed to avoid duplication.

**Comment by Cyborgscode** [comment]
A blog does need more than a main rss feed, i.e. a rss feed link that is tagged with one or more specific tag(s) or categories. So +1 for the FR.

**Comment by candidexmedia** [comment]
+1 for this. I also posted this request in the forum: https://forum.getpublii.com/topic/distinct-rss-feeds-for-each-tag

**Comment by s3nnet** [comment]
Yeah, I know... just posted there pointing out a workaround with pipes.digital
